obspy/imaging/waveform.py

```diff
@@ -1300,6 +1308,10 @@ def __sect_init_plot(self):
             else:
                 raise NotImplementedError("sect_orientiation '%s' is not "
                                           "valid." % self.sect_orientation)
+            if self.wiglets:
+                fillfun(time, data, self._tr_offsets_norm[_tr], where=data > self._tr_offsets_norm[_tr], facecolor='blue')
+                fillfun(time, data, self._tr_offsets_norm[_tr], where=data < self._tr_offsets_norm[_tr], facecolor='red')
```

Comment:
> And maybe we can start to deprecate the gazillion function parameters we have there.

:+1: but a lot of work..

> Maybe better to offer a way to reset the style?

Seems a much to me.. do we really need that much complexity..?

Or if we really need something like this.. maybe sum all those settings in one place, like matplotlib does it (with rcparams)?